src/shared/components/cells/CellCloneOverlay.tsx

```diff
@@ -0,0 +1,169 @@
+import React, {FC, useState, useContext, useEffect} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Context
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+// Clockface
+import {
+  Dropdown,
+  Button,
+  Overlay,
+  ComponentColor,
+  InputLabel,
+  SlideToggle,
+  Form,
+} from '@influxdata/clockface'
+
+// Actions
+import {createCellWithView, deleteCellAndView} from 'src/cells/actions/thunks'
+import {getOverlayParams} from 'src/overlays/selectors'
+import {getDashboardIDs} from 'src/dashboards/utils/getDashboardIds'
+import {getOrg} from 'src/organizations/selectors'
+
+// Types
+import {AppState} from 'src/types'
+
+// Metrics
+import {event} from 'src/cloud/utils/reporting'
+import {Dashboard} from 'src/client'
+
+const CellCloneOverlay: FC = () => {
+  const [otherDashboards, setOtherDashboards] = useState<Dashboard[]>([])
+  const dispatch = useDispatch()
+  const {id: orgID} = useSelector(getOrg)
+
+  const currentDashboardID = useSelector(
+    (state: AppState) => state.currentDashboard.id
+  )
+
+  useEffect(() => {
+    getDashboardIDs(orgID).then(retrievedBoards => {
```

Comment:
I think this might be overkill? We should be able to access the list of dashboards in a selector. Even if we deep link to a specific dashboard, I think we need to load ALL the dashboards, then the cells and views for an individual dashboard? I could be wrong